app/src/main/java/com/example/weather/screen/map/MapPresenter.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.weather.screen.map
+
+import com.example.weather.data.model.Weather
+import com.example.weather.data.repository.WeatherRepository
+
+class MapPresenter(
+    private val repository: WeatherRepository
+) : MapContract.Presenter {
+    override fun getWeather(latitude: Double, longitude: Double, isNetworkEnable: Boolean, isCurrent: Boolean) {
+        // todo later
```

Comment:
```suggestion
        // TODO implement later
```
mark để làm gì thì xem link sau sẽ rõ
https://stackoverflow.com/questions/23384804/how-to-see-all-todo-tasks-in-android-studio